client/package.json

```diff
@@ -59,6 +59,11 @@
                     "default": false,
                     "description": "Show or hide the status bar item asking you to report any PHP Intellisense bugs"
                 },
+                "crane.showStatusBarIcon": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
If we're using `true` as the default in code, it should be the default here as well